src/algebra/group/to_additive.lean

```diff
@@ -328,3 +378,9 @@ end to_additive
 
 /- map operations -/
 attribute [to_additive] has_mul has_one has_inv has_div
+/- the following types are supported by `@[to_additive]` and mapped to themselves. -/
+attribute [to_additive empty] empty
+attribute [to_additive pempty] pempty
+attribute [to_additive punit] punit
+attribute [to_additive unit] unit
+-- attribute [to_additive_ignore_args 21] times_cont_mdiff_map
```

Comment:
Can this line be removed since it is in `geometry/manifold/times_cont_mdiff_map.lean`?
```suggestion
```